tools/update-versions.js

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env node
+'use strict';
+
+const fs = require('fs-extra');
+const path = require('path');
+const root = path.join(__dirname, '..');
+
+const manifest = require(`${root}/manifest.json`);
+const pkg = require(`${root}/package.json`);
+
+const good = '\x1b[32m%s\x1b[0m';
+const warn = '\x1b[36m%s\x1b[0m';
+
+function exit(err) {
+  if (err) {
+    console.error(err);
+  }
+  process.exit(err ? 1 : 0);
+}
+
+function verToArray(v) {
+  return v.replace('v', '').split('.').map(Number);
+}
+
+// Simple compare function since we can't require semverCompare here
+function compare(v1, v2) {
```

Comment:
I think we can require the compare function with:
```js
const semverCompare = require("semver-bundle");
```

Reply:
I tried that... since the semver file isn't using `export`, the `semverCompare` variable is an empty object.

Reply:
I thought about reading in the file, then prepending a `exports.` to the string and then using `eval`, but that seemed like a bad idea... and it was just easier to create a simple compare function.

Reply:
That's weird. `semver-bundle` does export the compare function:
https://unpkg.com/semver-bundle@0.1.0/index.js
Did you required the dist file?

Reply:
BTW, there is a simple tool to sync the version number with package.json:
https://www.npmjs.com/package/sync-version

Reply:
Oh nice! Well, I wrote this because I know @tophf is used to updating the `manifest` version, so this script updates whichever file has the older version.

Reply:
And look at the [`semver.js`](https://github.com/openstyles/semver-bundle/blob/master/dist/semver.js), it starts out like this:

```js
var semverCompare=function(){"use strict";var s=256,i=Number.MAX_SAFE_INTEGER||9007199254740991,...
```